assets/js/index.js

```diff
@@ -0,0 +1,110 @@
+
+document
+    .querySelector("ul")
+    .addEventListener("click", e =>{
+        if(e.target.tagName === 'LI'){
+            if(e.metaKey){
+                e.target.remove();
+            }
+            else{
+                e.target.style = 'text-decoration : line-through';
+            }
+        }
+    });
+
+for(let butt of document.querySelectorAll("button")){
+    butt.addEventListener("click", e =>{
```

Comment:
Since each button is an individual controller for a distinct action, we should add event listeners to them individually, not in a loop.